src/containers/PatientProfilePage.tsx

```diff
@@ -41,6 +41,7 @@ const PatientProfilePage = ({
 }: {
   match: {
     params: {
+      eventId: string;
```

Comment:
is this used anywhere in this file?

Reply:
we can keep it I use the eventId in the transport flow pr